yaksh/templates/yaksh/courses.html

```diff
@@ -123,7 +133,7 @@
             </li>
             <br>
             <li>
-                <a href="{{URL_ROOT}}/exam/manage/duplicate_course/{{ course.id }}/">
+                <a class="test" href="{{URL_ROOT}}/exam/manage/duplicate_course/{{ course.id }}/" data-toggle="tooltip" title="Creates Copy of selected Course as well as its Modules, Lessons/Quizzes" data-placement="top">
```

Comment:
I am confused by the nomenclature of  the cloned course. Both deep and shallow cloning of courses are called Clone course. Should it be named differently? Also once a cloned copy is created, the course name is called Copy of *course*, irrespective of how it was cloned.